docs/components/modeler/web-modeler/collaboration.md

```diff
@@ -30,8 +32,16 @@ Additionally, the **Owner** of the organization has special privileges to do adm
 
 #### Super-user mode
 
-Super-user mode is only available to the **Owner** of the organization. The main purpose of this mode is to assign collaborators
-to orphaned projects (which have no collaborators). Ordinarily, these projects would not be accessible or visible to any users.
+:::note
+Super-user mode is not yet available in Web Modeler Self-Managed.
+:::
+
+Super-user mode is only available to the **Owner** of the organization and can be enabled via the user menu in Web Modeler:
+
+<p><img src={SuperUserModeImg} style={{width: 280}} alt="Enable super-user mode in Web Modeler's user menu" /></p>
```

Comment:
FYI: I wrapped the image in a `<p>` tag to get a margin below the image.

Reply:
@pepopowitz formatting/style comment here you may want to look at.

Reply:
When we use markdown images (`![...](...)`), images get automatically wrapped in `p` tags...so it makes sense to me that if you used an HTML image, you'd also need to add the `p`. 👍

Reply:
Thanks @pepopowitz. By the way, I tried Markdown first, but the image was displayed too big and blurry. So I used HTML to be able to set the correct size.